### DISCUSSION TITLE
Feature request: Ability to disable 'update date and time' popup

### DISCUSSION THREAD

**Discussion by Herbis** [gopro/labs#675] [Feature Requests]
### Motivation
Every time you switch batteries ( or have not had a battery in your GoPro for a while) or maybe you just use external power source that you connect / disconnect, the camera will show an 'update date and time' popup, which you can't dismiss (without just starting a recording). This is a very annoying, as I often have to quickly setup the camera, adjust the view, and move on to the next camera. This prevents me from doing that. 
I also do not care what date is set on my video footage. It's okay for it to use the year 1970 for all I care, and this is the case for plenty of other customers, that are in similar predicament. 

### Expected behavior
1. Preferred - popup is never shown. Permanent setting enabled with QRControl.
2. Alternate - popup can be dismissed with one click (have an 'ignore' button or something). Permanent setting enabled with QRControl, or default firmware behavior. 

**Requesting for GoPro camera models:** Hero 11, Hero 10, future Hero models

Here are some reddit and GoPro forum posts looking for the same features, referencing lost footage and frustration:
https://community.gopro.com/s/question/0D53b00008BtUFiCAN/how-to-disable-update-date-time-popup-every-time-hero-8-boots
https://community.gopro.com/s/question/0D53b00008BtgrXCAR/disable-date-time-prompt?language=en_US
https://www.reddit.com/r/gopro/comments/kf5qau/how_to_disable_update_date_and_time_popup_when/
https://www.reddit.com/r/gopro/comments/164epcx/does_anyone_else_find_it_annoying_when_you_switch/

**Comment by dnewman-gpsw** [comment]
It exists, look in https://gopro.github.io/labs/control/extensions/ under BYPS

![image](https://github.com/gopro/labs/assets/10856084/5d6832da-9308-4b2f-a37e-2cdbe5d8fdae)

**Reply by Herbis** [reply]
Thank you, wasn't aware of such setting!

I've tested it and it has a significant / deal-breaking downside, it affects popups for shortcuts, like if I want to change zoom or lense width, and with video profiles, if I want to adjust them through edit, the popup immediately closes. Is this a know issue / limitation? 

> Remember to set time and date if you remove the battery.

Is this a recommendation, or a requirement?

**Reply by dnewman-gpsw** [reply]
I think you have to two posts mixed up.  The boot script in the other post is causing the shortcuts not to work.  BYPS is not at fault, and is working fine.

**Reply by Herbis** [reply]
I see, that's likely true, I thought after disabling this setting it I had better experience with the shortcuts, but seems I was wrong.  Thanks for pointing that out.

**Comment by dnewman-gpsw** [comment]
moved to discussions, as **Issues** postings get closed when solved.
